src/lib.js

```diff
@@ -886,6 +890,7 @@ export const getItems = async (
   let hotm = "mining_core" in profile ? await getHotmItems(profile, packs) : [];
 
   output.armor = armor.filter((x) => x.rarity);
+  output.equipment = equipment.filter((x) => x.rarity);
```

Comment:
what does `.filter((x) => x.rarity)` do?

Reply:
I saw there was some discussion with dukio about this but I didn't see the resolution. What did this do again?

Reply:
> I saw there was some discussion with dukio about this but I didn't see the resolution. What did this do again?

It fixes equipment being sorted wrongly

Reply:
Basically we need to filter out null items otherwise it breaks the all items Map in stats defer and other issues too iirc